src/bin/alerts.py

```diff
@@ -0,0 +1,331 @@
+from json import dumps
+from typing import List, Optional
+from uuid import UUID
+
+import typer
+from zanshinsdk import (
+    AlertSeverity,
+    AlertsOrderOpts,
+    AlertState,
+    Client,
+    Languages,
+    SortOpts,
+)
+from zanshinsdk.alerts_history import FilePersistentAlertsIterator
+from zanshinsdk.following_alerts_history import FilePersistentFollowingAlertsIterator
+
+import src.config.sdk as sdk_config
+from src.lib.models import AlertStateSetable
+from src.lib.utils import output_iterable
+
+app = typer.Typer()
+
+
+@app.command(name="list")
+def alert_list(
+    organization_id: UUID = typer.Argument(..., help="UUID of the organization"),
+    scan_target_id: Optional[List[UUID]] = typer.Option(
+        None, help="Only list alerts from the specified" "scan targets."
```

Comment:
Why open and close quotes?
Also, no need for the final period (".") to be consistent with others.